lib/themes/dosomething/paraneue_dosomething/js/lab.js

```diff
@@ -15,10 +15,7 @@ $(document).ready(function() {
   if (Drupal.settings.dsOnboarding.enabled) {
     const jsxElementHook = 'jsx-onboarding';
     const slides = [ContextSlide, ContextSlide] // @TODO: create other slide components.
-    const campaign = {
-      // @TODO: switch to use Drupal.settings.dsCampaign once it is available.
-      title: 'Awesome Campaign Placeholder'
-    }
+    const campaign = Drupal.settings.dsCampaign || null;
```

Comment:
I agree. `undefined` just means a variable was declared but is not assigned yet. I'm specifically assigning a `null` value to indicate there is no campaign data available.